### PR TITLE
First Commit: comments are hidden and I made it that way and I would …

### PR DESCRIPTION
**Pull request by hewford**
comments are hidden and I made it that way and I would like to reverse that now. Currently you have to push comments to show them

### CODE REVIEW

**Comment by hewford** [comment]
Basically, when you click the post. it hides everything else. When you click comments, it shows that posts comments. Only difference

**Comment by hewford** [comment]
I updated javascript and html to use handlebar templates. Also, I removed the hover event in favor of targeting closest parent. 
(I still need to delete from posts and also have posts render the page each time a change is made)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        <div class="comments-padding">
+        </div>
+        <p class="col-10 comment-message">{{ commentMessage }}</p>
+      </div>
+    </div>
+  </div>
+</script>
+
+
+<script type="text/javascript" src="postTemplates.js"></script>
```
Comment (amhayslip) [inline index.html]: Think of re-naming this - there are not templates in this file

File: initialize.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+var STORAGE_ID = 'project-distillery';
+
+var saveToLocalStorage = function () {
+localStorage.setItem(STORAGE_ID, JSON.stringify(posts));
```
Comment (amhayslip) [inline initialize.js]: indenting

File: initialize.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+var getFromLocalStorage = function () {
+  return JSON.parse(localStorage.getItem(STORAGE_ID) || '[]');
```
Comment (amhayslip) [inline initialize.js]: indenting

File: initialize.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+var commentCount=0; //counts amount of comments made
```
Comment (amhayslip) [inline initialize.js]: `var commentCount = 0`

File: initialize.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+//If we want to load the page without any pre-loaded content, simply remove newPost(), newComment, and thisPostId from this page. And delete the object in the area
+
+console.log("initial variables are created")
+console.log("posts array already contains initial object --> pushing to post skipped")
+
+
+$( document ).ready(function() {
+
+  let thisPostId = 1; //since the page adds the first post object, I initialized thisPostId since there is no hovering that happens to find the variable.
```
Comment (amhayslip) [inline initialize.js]: hovering?

File: initialize.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+//If we want to load the page without any pre-loaded content, simply remove newPost(), newComment, and thisPostId from this page. And delete the object in the area
+
+console.log("initial variables are created")
+console.log("posts array already contains initial object --> pushing to post skipped")
+
+
+$( document ).ready(function() {
```
Comment (amhayslip) [inline initialize.js]: `$(document).ready`

File: postTemplates.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+            posts private).
+        ---Currently I had more ids in my HTML than I need
+            Started to use .closest() and .sibling(), but
+            need to finish the work.
+        ---Something is wrong with saveToLocalStorage()
+            were I sometimes have to remove a post twice
+        ====================================================
+*/
+var renderPosts = function(){
+  let postCheck = getFromLocalStorage();
```
Comment (amhayslip) [inline postTemplates.js]: `postCheck`? I'd consider renaming

File: postTemplates.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+      }],
+    }]
+  }
+  else{
+    posts = getFromLocalStorage();
+  }
+  $('.posts').empty();
+
+  for (var postCount = 0; postCount < posts.length; postCount+= 1){
+
```
Comment (amhayslip) [inline postTemplates.js]: No new line

File: templates.html [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<script id="post-template" type="text/x-handlebars-template">
```
Comment (amhayslip) [inline templates.html]: If you never used this file, you can remove it

**Pull request closed without merging**